components/hooks/post/postHooks.ts

```diff
@@ -61,6 +66,10 @@ const usePost = (commentId: string | null): UsePost => {
     };
   }, []);
 
+  useEffect(() => {
```

Comment:
We don't need to have two useEffects here, we can just use one. I'll replace the current one that has an empty dep array and we will keep this one. (The one with the empty array has a useless cleanup function, not sure why I had that there, but we will just get rid of the whole thing)